.gitignore

```diff
@@ -2,3 +2,4 @@
 _site
 .sass-cache
 .jekyll-metadata
+*~
```

Comment:
The point was to avoid tracking the gedit temporary files, these files' name being _$filename_~, it seems necessary to set this value.

Maybe you prefer `**/*~` but the result seems to be the same.

Is that what you had in mind?

Reply:
Not really, I've just want to point you out that `*~` and `**/*~` do not produce the same result.
- `*~` will only ignore temporary files at the directory from which the `.gitignore` file is located
- `**/*~` will ignore any temporary files from any directory (then subdirectories as well) starting from where the `.gitignore` file is located.

So if your initial request would be to ignore any temporary files from any directory, you should prefer to use `**/*~`

Reply:
Actually `*~` is woking in the same way since there is no `/` in the described path.

See the [documentation](https://git-scm.com/docs/gitignore#_pattern_format), the 6th and 7th point ;)

Reply:
My bad, there is effectively no slash in the path so it is working not only in the current directory.
Ok, you can forget this change request ;-)